media-libs/netpbm/netpbm-10.86.22.ebuild

```diff
@@ -112,6 +112,7 @@ src_prepare() {
 			-e 's:(pamrgbatopng|pngtopnm).*::' \
 			test/legacy-names.{ok,test} || die
 		sed -i -e '/^$/d' test/legacy-names.ok || die
+		sed -i -e 's:png-roundtrip.*::' test/Test-Order || die
```

Comment:
`if ! use png; then`? 
Just let me know if it's supposed to work **when** png is enabled.

Reply:
Without `png` USE flag, tools like `pamtopng` and `pngtopam` are not built, so we need to disable the tests using them.  So `if ! use png` I remove them with `sed`.  If png is enabled, the tests run fine.